frontend/packages/console-shared/src/selectors/node.ts

```diff
@@ -18,6 +18,11 @@ export const getNodeRoles = (node: NodeKind): string[] => {
   );
 };
 
+export const isLabelPresent = <A extends K8sResourceKind = K8sResourceKind>(
```

Comment:
If we are expecting the k8sResourceKind object to be manipulated/added new property. It will not have any harm but I think we can keep it simple for now.